apps/systemtags/src/components/SystemTagsCreationControl.vue

```diff
@@ -10,13 +10,13 @@
 		</h4>
 
 		<p class="settings-hint">
-			{{ t('settings', 'If enabled, regular accounts will be restricted from creating new tags but will still be able to assign and remove them from their files.') }}
+			{{ t('settings', 'If enabled, regular accounts will be restricted from editing or creating new tags but will still be able to assign and remove them from their files.') }}
 		</p>
 
 		<NcCheckboxRadioSwitch type="switch"
 			:checked.sync="systemTagsCreationRestrictedToAdmin"
 			@update:checked="updateSystemTagsDefault">
-			{{ t('settings', 'Restrict tag creation to admins only') }}
+			{{ t('settings', 'Restrict tag edition and creation to admins only') }}
```

Comment:
I think it is

```suggestion
			{{ t('settings', 'Restrict tag editing and creation to admins only') }}
```

Reply:
I checked and you're right! What a weird phrazing :grin:
I would even go for 
```suggestion
			{{ t('settings', 'Restrict tag creation and editing to administrators only') }}
```